### PR TITLE
EVG-3515 (feat) Trendchart - Change point statistics

### PR DESCRIPTION
**Pull request by unrealsolver**
trendchart-cp1
trendchart-cp2
trendchart-3

### CODE REVIEW

File: public/static/plugins/perf/js/trend_chart.js [hunk 1]
```diff
@@ -626,17 +678,53 @@ var drawSingleTrendChart = function(params) {
...
-  var changePointsG = chartG.append('g')
-    .attr('class', 'g-change-points')
-
   function redrawChangePoints() {
     // Render change points
-    var changePoints = changePointsG.selectAll('g.change-point')
-      .data(_.filter(changePointForLevel, function(d) {
-        return d.level.isActive
-      }))
+    var activePoints = _
```
Comment (john-m-liu) [inline public/static/plugins/perf/js/trend_chart.js]: This isn't used

File: public/static/plugins/perf/js/trend_chart.js [hunk 2]
```diff
@@ -626,17 +678,53 @@ var drawSingleTrendChart = function(params) {
...
+          return yScale(getValueFor(d.level)(series[d.changePoint._meta.lastRevIdx]))
+        },
+      })
+      .style({
+        stroke: 'red',
+        'stroke-width': '4',
+      })
+
+    changePointSegments
+      .on('mouseover', function(d) {
```
Comment (john-m-liu) [inline public/static/plugins/perf/js/trend_chart.js]: What's the purpose of this?

**Comment by unrealsolver** [comment]
Please don't merge - I have to made few changes due to changes in changePoint format introduced by Jim

**Comment by unrealsolver** [comment]
Readu for merge!

**Pull request merged**